lighthouse-viewer/test/viewer-test-pptr.js

```diff
@@ -134,7 +134,8 @@ describe('Lighthouse Viewer', function() {
     const auditErrors = await viewerPage.$$eval(errorSelectors, getErrors, selectors);
     const errors = auditErrors.filter(item => item.explanation.includes('Audit error:'));
     const unexpectedErrrors = errors.filter(item => {
-      return !item.explanation.includes('Required RobotsTxt gatherer did not run');
+      return !item.explanation.includes('Required RobotsTxt gatherer did not run') &&
```

Comment:
I don't understand what the proper fix for this would be, but from the discussion in the channel I understood it might not be easy and we're fine just ignoring the problem.

Reply:
Yup, ignoring is what we wanted here for now :)